lib/browserGlobals.js

```diff
@@ -0,0 +1,8 @@
+// This defines globals that will be used in the browser context
```

Comment:
I think the filename here should be `browser-globals.js` to match the naming convention of the other files in this project.